### PR TITLE
add chart for cerebro

### PR DESCRIPTION
**Pull request by davidkarlsen**
This is a chart for the image: https://github.com/yannart/docker-cerebro which contains the Cerebro application: https://github.com/lmenezes/cerebro
which is an open source(MIT License) elasticsearch web admin tool built using Scala, Play Framework, AngularJS and Bootstrap.

---
This change is [Reviewable](https://reviewable.io/reviews/kubernetes/charts/6291)

### CODE REVIEW

File: incubator/cerebro/OWNERS [hunk 1]
```diff
@@ -0,0 +1,4 @@
+apporvers:
```
Comment (timstoop) [inline incubator/cerebro/OWNERS]: approvers

File: incubator/cerebro/README.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
+# Cerebro
+
+Cerebro is an open source(MIT License) elasticsearch web admin tool built using Scala, Play Framework, AngularJS and Bootstrap.
```
Comment (timstoop) [inline incubator/cerebro/README.md]: Please add a space after "open source".

**Review by timstoop — CHANGES_REQUESTED** [review]
I actually use Cerebro quite a lot, but we use it directly from the desktop instead. Nice addition!

**Comment by davidkarlsen** [comment]
@timstoop fixed

File: incubator/cerebro/requirements.yaml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+dependencies:
+  - name: common
```
Comment (unguiculus) [inline incubator/cerebro/requirements.yaml]: I'd suggest not to use the common chart. It's future is kind of unclear, plus, it is in incubator and will stay there. Usually, I suggest to move charts directly to stable, but stable charts are not allowed to depend on incubator charts.

**Review by unguiculus — CHANGES_REQUESTED** [review]

File: incubator/cerebro/requirements.yaml (hunk 3 above)
Comment (davidkarlsen) [reply incubator/cerebro/requirements.yaml]: oh no - I liked the style of less typing - maybe some notes should be put into it to avoid it's usage?

Comment (davidkarlsen) [reply incubator/cerebro/requirements.yaml]: @unguiculus fixed

**Comment by davidkarlsen** [comment]
What is left to do with this one?

File: stable/cerebro/templates/deployment.yaml [hunk 4]
```diff
@@ -0,0 +1,51 @@
+apiVersion: apps/v1beta2
```
Comment (unguiculus) [inline stable/cerebro/templates/deployment.yaml]: Use `apps/v1`

**Review by unguiculus — CHANGES_REQUESTED** [review]

File: stable/cerebro/templates/deployment.yaml (hunk 4 above)
Comment (davidkarlsen) [reply stable/cerebro/templates/deployment.yaml]: fixed

**Review by unguiculus — APPROVED** [review]

**Pull request merged**